vignettes/why-it-works.Rmd

```diff
@@ -0,0 +1,190 @@
+---
+title: "Interval censoring and truncation in epidemiological delay modelling"
+format: html
+bibliography: library.bib
+---
+
+# Introduction {#introduction}
+
+Time to event analysis, also known as survival analysis, concerns estimating the distribution of delay times between events. A distinctive feature of the field are the methodological techniques used to deal with the missing data problems common in data sets of delay times [@leung1997censoring]. In `primarycensoreddist` we focus on two particular missing data problems:
```

Comment:
Here and later could do some referencing out to Park and Charniga for alternative explanations of various points.

Reply:
Yeah got that in there.